src/App.tsx

```diff
@@ -46,11 +46,7 @@ function App() {
         path="/events/:eventId/edit"
         component={(props) => <EventCreationPage mode="edit" {...props} />}
       />
-      <Route
-        exact
-        path="/manage/:resource"
-        component={ResourceOverviewPage}
-      />
+      <Route exact path="/manage/:resource" component={ResourceOverviewPage} />
```

Comment:
This is probably a prettify change. Are we still good with it in this PR?

Reply:
yes - this means that whoever checked this in forgot to run prettier before merging and if we don't address this here then it'll keep showing up on other people's PRs (after they run prettier)